### PR TITLE
✨(models) add peer instruction events pydantic models

### PR DESCRIPTION
**Pull request by wilbrdt**
## Purpose

Peer instruction events UBC events generated during platform usage.
They are emitted by the server to capture information about
peer instruction XBlock.

## Proposal

Write `pydantic` models, edge-case and selector tests for:

- [x] ubc.peer_instruction.accessed
- [x] ubc.peer_instruction.original_submitted
- [x] ubc.peer_instruction.revised_submitted

### CODE REVIEW

File: tests/models/edx/peer_instruction/test_statements.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    assert model is class_
+
+
+@custom_given(PeerInstructionAccessed)
+def test_models_edx_peer_instruction_accessed_with_valid_statement(
+    statement,
+):
+    """Tests that a `ubc.peer_instruction.accessed` statement has the expected
+    `event_type`."""
+    assert statement.event_type == "ubc.peer_instruction.accessed"
```
Comment (quitterie-lcs) [inline tests/models/edx/peer_instruction/test_statements.py:42]:
```suggestion
    assert statement.event_type == "ubc.peer_instruction.accessed"
    assert statement.name == "ubc.peer_instruction.accessed"
```

File: tests/models/edx/peer_instruction/test_statements.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+
+@custom_given(PeerInstructionOriginalSubmitted)
+def test_models_edx_peer_instruction_original_submitted_with_valid_statement(
+    statement,
+):
+    """Tests that a `ubc.peer_instruction.original_submitted` statement has the
+    expected `event_type`."""
+    assert statement.event_type == "ubc.peer_instruction.original_submitted"
```
Comment (quitterie-lcs) [inline tests/models/edx/peer_instruction/test_statements.py:51]:
```suggestion
    assert statement.event_type == "ubc.peer_instruction.original_submitted"
    assert statement.name == "ubc.peer_instruction.original_submitted"
```

File: tests/models/edx/peer_instruction/test_statements.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+
+@custom_given(PeerInstructionRevisedSubmitted)
+def test_models_edx_peer_instruction_revised_submitted_with_valid_statement(
+    statement,
+):
+    """Tests that a `ubc.peer_instruction.revised_submitted` statement has the
+    expected `event_type`."""
+    assert statement.event_type == "ubc.peer_instruction.revised_submitted"
```
Comment (quitterie-lcs) [inline tests/models/edx/peer_instruction/test_statements.py:60]:
```suggestion
    assert statement.event_type == "ubc.peer_instruction.revised_submitted"
    assert statement.name == "ubc.peer_instruction.revised_submitted"
```

**Review by quitterie-lcs — APPROVED** [review]
Let's go !
